docs/src/puppeteer-js.md

```diff
@@ -0,0 +1,110 @@
+---
+id: puppeteer
+title: "Migrating from Puppeteer"
+---
+
+<!-- TOC -->
+
+## Migration Principles
+
+This guide describes migration to [Playwright Library](./library) from Puppeteer. The APIs have similarities, but Playwright offers much more possibilities for web testing and cross-browser automation.
+
+- Most Puppeteer APIs can be used as is
+- Playwright is cross-browser
+- You probably don't need explicit wait
+
+## Cheat Sheet
+
+| Puppeteer                                          | Playwright Library                          |
+|----------------------------------------------------|---------------------------------------------|
+| `await puppeteer.launch(...)`                      | `await playwright.chromium.launch(...)`     |
```

Comment:
consider describing how to start a specific browser (firefox / chromium) and maybe noting that webkit is only supported in playwright